rfcs/interop_2023.md

```diff
@@ -0,0 +1,84 @@
+# RFC 116: Interop 2023
+
+## Summary
+
+Interop 2023 is an effort to improve interoperability in areas believed to be important to web developers and users. This is the next iteration of [Interop 2022](https://github.com/web-platform-tests/interop-2022), an effort which is still ongoing at the time of this RFC.
+
+The focus areas will be selected by a proposals period followed by a proposals review and selection process. The [Interop 2023 team](https://github.com/orgs/web-platform-tests/teams/interop-2023) will manage this process and make decisions based on consensus, see [governance](#governance).
```

Comment:
I do vaguely wonder if it's worthwhile having a per-year repo? To me it doesn't seem super worthwhile.

Reply:
I take it the proposal is to redirect https://github.com/web-platform-tests/interop-2022 to https://github.com/web-platform-tests/interop?

Reply:
As discussed in https://github.com/web-platform-tests/interop-2022/issues/95 this will be a separate RFC, in parallel to this one.